docs/kafka/getting-started-kafka/README.adoc

```diff
@@ -228,24 +217,24 @@ ifdef::qs[]
 endif::[]
 ifndef::qs[]
 . Verify that the bootstrap server, client credentials, and authentication token endpoint are saved to a secure location.
-. Verify that your service account was successfully created in the *Service Accounts* page.
+. Verify that your service account was successfully created on the *Service Accounts* page.
 endif::[]
 
 [id="proc-setting-service-account-permissions_{context}"]
 == Setting permissions for a service account in a Kafka instance in {product-kafka}
 
 [role="_abstract"]
-After you create a service account to connect to a Kafka instance, you must also set the appropriate level of access for that new account in the Access Control List (ACL) of the Kafka instance. {product-kafka} uses ACLs provided by Kafka that enable you to manage how other user accounts and service accounts are permitted to interact with the Kafka resources that you create.
+After you create a service account to connect to a Kafka instance, you must also set the appropriate level of access for that new account in the Access Control List (ACL) of the Kafka instance. {product-long-kafka} uses ACLs provided by Kafka that enable you to manage how other user accounts and service accounts are permitted to interact with the Kafka resources that you create.
 
 .Prerequisites
-* You've created a Kafka instance and the instance is in the *Ready* state.
+* You have a running Kafka instance in {product-kafka}.
 * You've created a service account that you want to allow to access the running Kafka instance.
 
 .Procedure
-. In the *Kafka Instances* page of the web console, click the name of the Kafka instance that you want the service account to access.
+.  In the {product-long-kafka} {service-url-kafka}[web console^], select *Kafka Instances* and then click the name of the Kafka instance that you want the service account to access.
 . Click the *Access* tab to view the current ACL for this instance.
-. Click *Manage access*, use the *Account* drop-down menu to select the service account that you previously created, and click *Next*.
-. Under *Assign Permissions*, use the drop-down menu to select the *Consume from a topic* and the *Produce to a topic* permission options, and set all resource identifiers to `Is` and all identifier values to `*`.
+. Click *Manage access*, use the *Account* list to select the service account that you previously created, and click *Next*.
```

Comment:
I'd suggest breaking this step into 2 as after they click Manage access, a new dialog box opens so it's not really part of the same step.

Reply:
Yes, good call.